velox/exec/SetAccumulator.h

```diff
@@ -106,10 +153,50 @@ struct SetAccumulator {
                                  : uniqueValues.size();
   }
 
+  void serializeNullIndex(char* buffer) {
+    auto nullIndexValue =
+        nullIndex.has_value() ? nullIndex.value() : kNoNullIndex;
+    memcpy(buffer, &nullIndexValue, kVectorSizeT);
+  }
+
+  /// Extracts in result[index] a serialized VARBINARY for the Set Values.
+  /// This is used for the spill of this accumulator.
+  void serialize(const VectorPtr& result, vector_size_t index) {
+    // The serialized value is the nullOffset (kNoNullIndex if no null is
+    // present) followed by the unique values in order of their indices.
+    // The null position is skipped when serializing the values.
+    size_t totalBytes = kVectorSizeT + kValueSizeT * uniqueValues.size();
```

Comment:
kVectorSizeT and kValueSizeT names are inconsistent with the rest of the code base; why not use sizeof(vector_size_t) and size(T) explicitly?

Reply:
There were too many places to repeat sizeof(...) in the code, so I made variables for them. I've renamed them for readability. Let me know if you prefer other naming.